gui/src/main/java/org/jboss/as/console/client/core/UIMessages.properties

```diff
@@ -53,6 +53,7 @@ deploymentInUse={0} is in use. Please remove its assignments first.
 deploymentSuccessfullyDeployed={0} successfully deployed.
 deploymentSuccessfullyReplaced={0} successfully replaced.
 disableConfirm=Disable {0} ?
+disableSSOOnSecurityDomainConfirm=Disable Single Sign-On on application security domain '{0}'?
```

Comment:
There is a GWT build error related to the quotation marks, can you remove it or add a double quotation marks ?
The GWT error is: 
[ERROR] Required argument 0 not present: Disable Single Sign-On on application security domain '{0}'?